packages/cli/src/Generate.ts

```diff
@@ -49,9 +47,10 @@ ${chalk.bold('Usage')}
 
 ${chalk.bold('Options')}
 
-  -h, --help   Display this help message
-    --schema   Custom path to your Prisma schema
-     --watch   Watch the Prisma schema and rerun after a change
+    -h, --help   Display this help message
+      --schema   Custom path to your Prisma schema
+  --data-proxy   Enable the Data Proxy in the Prisma Client
```

Comment:
New way to enable `--data-proxy`. There will be a followup later to also enable this for `prisma studio`.